CHANGELOG.md

```diff
@@ -7,10 +7,31 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## 0.1.0 - 2019-03-26
+### Added
+
+- Summary status for facility lists [#378](https://github.com/open-apparel-registry/open-apparel-registry/pull/378)
```

Comment:
With long links such as these, it would be hard to wrap this file to be within 80 characters, but wanted to run it by you @kellyi.